packages/@aws-cdk/aws-iotevents/lib/expression.ts

```diff
@@ -0,0 +1,75 @@
+import { IInput } from './input';
+
+/**
+ * Expression for events in Detector Model state
+ * @see https://docs.aws.amazon.com/iotevents/latest/developerguide/iotevents-expressions.html
+ */
+export abstract class Expression {
```

Comment:
> to present an smart API for conditions as mapping-tenplate.ts in aws-appsync instead of just string

is here!

Usage:
```ts
// meaning `condition: 'currentInput("test_input") && $input.test_input.payload.temperature == 31.5'`
condition: iotevents.Expression.and(
  iotevents.Expression.currentInput(input),
  iotevents.Expression.eq(
    iotevents.Expression.inputAttribute(input, 'payload.temperature'),
    iotevents.Expression.fromString('31.5'),
  ),
)
```